packages/block-editor/src/components/inserter/quick-inserter.js

```diff
@@ -138,8 +138,7 @@ export default function QuickInserter( {
 
 			{ setInserterIsOpened && (
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
I have removed the custom CSS for height as it was overriding the default size styles. Let me know if any further changes are needed.

Thanks 🙇 

| Before | After     |
| :-------- | :------- |
| ![quick-inserter-before](https://github.com/user-attachments/assets/64f7bbad-8821-4653-909d-74fd480da364) | ![quick-inserter-after](https://github.com/user-attachments/assets/f611744a-e950-4028-81ed-fee3617f174f) |